README.md

```diff
@@ -12,6 +12,24 @@ conda activate yeat
 pip install .
 ```
 
+### Bandage
+
+In order to run [`Bandage`](https://github.com/rrwick/Bandage) in YEAT's workflow, users will need to manually download the pre-built binaries files [here](https://rrwick.github.io/Bandage/) or build it from the source code [here](https://github.com/rrwick/Bandage). Instructions for building Bandage from the source code can be found [here](https://github.com/rrwick/Bandage#building-from-source).
+
+If the pre-built binary files are used, users will need to update their environment's `$PATH` to include the path of the directory containing the binary file.
+
+Example for Ubuntu:
+```
+export PATH={BANDAGE_DIR}:$PATH
+```
+
+Example for MacOS:
+```
+export PATH={BANDAGE_DIR}/Bandage.app/Contents/MacOS:$PATH
+```
+
+In order to run the pre-built binary files successfully, ensure that the binary file is kept in the same directory with all of the other files that came with it.
+
```

Comment:
Installation instructions for Bandage was added here.

Examples to update the user's $PATH is up for changes if needs be.

Not sure what else to say or grab's the user's attention. Hopefully the last line is informative because it took me a while to realize that each of the binary files requires various other files within the same dir.